package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "koa-gu",
-  "version": "0.0.16",
-  "description": "some small reusable modules for guardian koa micro-sites",
+  "name": "@guardian/koa-gu",
```

Comment:
This branch needs rebasing against `master` following https://github.com/guardian/koa-gu/pull/1/files.